woocommerce/woocommerce
# [Experimental] Product filters/remove template parts

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/51780

This PR removes the template parts for the Product Filters and Product Filters Overlay. In addition to this:

- Remove overlay settings from Product Filters block
- Make Product Filters block insertable via the inserter

Note the reimplementation of the overlay and overlay navigation settings will be handled in this issue: https://github.com/woocommerce/woocommerce/issues/51781



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

> [!NOTE]
> **Test by developers only**
>
> **Pre-req:**
> - Ensure you have the WC Beta Tester Plugin installed and go to Tools > WCA Test Helper > Features. Toggle on the experimental-blocks option.
>  - Reset Product Catalog Template.

1. Go to Appearance > Editor > Patterns > General
2. Ensure you no longer see Product Filters Template Part and Product Filters Overlay Template Part
3. Go to Appearance > Editor > Templates > WooCommerce > Product Catalog
4. Add the Product Filters block using the inserter
5. Select Product Filters (Experimental) in the List View and go to the Inspector Controls > Styles tab
6. Ensure you no longer see any Overlay controls
7. Save the template and go to frontend Shop page
8. Ensure you see the Product Filters block displayed



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Experimental block - remove template parts
</details>
